ci-operator/jobs/openshift/builder/openshift-builder-presubmits.yaml

```diff
@@ -63,3 +63,204 @@ presubmits:
+      serviceAccountName: ci-operator
+      volumes:
+      - configMap:
+          name: prow-job-cluster-launch-e2e
+        name: job-definition
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-gcp
+          - configMap:
+              name: cluster-profile-gcp
+    trigger: ((?m)^/test( all| gcp),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/gcp-crio
+    decorate: true
+    name: pull-ci-origin-e2e-gcp-crio
+    rerun_command: /test gcp-crio
+    run_if_changed: ^(vendor/k8s.io/kubernetes/pkg/kubelet)|^(pkg/build/builder)
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --secret-dir=/usr/local/e2e-gcp-crio-cluster-profile
+        - --template=/usr/local/e2e-gcp-crio
+        - --target=e2e-gcp-crio
+        command:
+        - ci-operator
+        env:
+        - name: TEST_FOCUS
+          value: Suite:openshift/conformance/parallel
+        - name: TEST_FOCUS_SERIAL
+          value: Suite:openshift/registry/serial
+        - name: JOB_NAME_SAFE
+          value: e2e-gcp-crio
+        - name: CLUSTER_TYPE
+          value: gcp
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: master.yaml
+              name: ci-operator-openshift-origin
+        - name: OVERRIDE_SPEC
+          value: '{"test_base_image": {"cluster": "", "namespace": "ci", "name": "src-cache-origin",
+            "tag": "master"}}'
+        image: ci-operator:latest
+        name: test
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-gcp-crio
+          name: job-definition
+          subPath: cluster-launch-e2e.yaml
+        - mountPath: /usr/local/e2e-gcp-crio-cluster-profile
+          name: cluster-profile
+      serviceAccountName: ci-operator
+      volumes:
+      - configMap:
+          name: prow-job-cluster-launch-e2e
+        name: job-definition
+      - name: cluster-profile
+        projected:
+          sources:
+          - secret:
+              name: cluster-secrets-gcp
+          - configMap:
+              name: cluster-profile-gcp-crio
+    trigger: ((?m)^/test( gcp-crio),?(\s+|$))
+  - agent: jenkins
+    always_run: true
+    branches:
+    - master
+    context: ci/openshift-jenkins/extended_conformance_install
+    labels:
+      master: ci.openshift.redhat.com
+    name: test_pull_request_origin_extended_conformance_install
+    rerun_command: /test extended_conformance_install
+    trigger: ((?m)^/test( all| extended_conformance_install),?(\s+|$))
+  - agent: jenkins
+    always_run: false
+    branches:
+    - master
+    context: ci/openshift-jenkins/extended_builds
+    labels:
+      master: ci.openshift.redhat.com
+    name: test_pull_request_origin_extended_builds
```

Comment:
Is this appropriate? How does that job get Origin code if that's not the repo under test?

Reply:
i'm looking for you guys to tell me.

I want to run the origin tests using the origin-docker-builder image produced from the builder repo.

Reply:
Give us like a week and you'll check in maybe one or two lines of config instead of fifty per job. You should be able to do this with the `cluster-install-src` workflow

Reply:
Don't include any Jenkins jobs.  You can't run them against ci-operator input anyway.

I have a replacement for the jenkins build job, it's just the GCP job with a different focus.  You need to qualify the e2e-gcp to be e2e-gcp-builds (note how crio does it)

Reply:
where is that job defined?

Reply:
Again if we can wait like a tiny bit Ben can add two lines of config to the ci-op and get that job generated versus spending N hours understanding all that crap right now

Reply:
i'll sit on it for a bit, but this is going to be blocking me pretty soon (arguably it's blocking me now because i was trying to get this lined up while doing some other things).